cmd/operator-sdk/execentrypoint/ansible.go

```diff
@@ -51,15 +51,23 @@ should use "run --local" instead.`,
 	return runAnsibleCmd
 }
 
-// setAnsibleRolePathEnvVar will set the default role path for the ANSIBLE_ROLES_PATH
-func setAnsibleRolePathEnvVar(flags *aoflags.AnsibleOperatorFlags) error {
-	if flags != nil && len(flags.AnsibleRolesPath) > 0 {
-		if err := os.Setenv(aoflags.AnsibleRolesPathEnvVar, flags.AnsibleRolesPath); err != nil {
-			return fmt.Errorf("failed to set %s environment variable: (%v)", aoflags.AnsibleRolesPathEnvVar, err)
+// setAnsibleEnvVars will set Ansible-defined environment variables from CLI flags.
+func setAnsibleEnvVars(flags *aoflags.AnsibleOperatorFlags) error {
+	if flags != nil {
+		if len(flags.AnsibleRolesPath) > 0 {
+			if err := os.Setenv(aoflags.AnsibleRolesPathEnvVar, flags.AnsibleRolesPath); err != nil {
+				return fmt.Errorf("failed to set %s environment variable: (%v)", aoflags.AnsibleRolesPathEnvVar, err)
+			}
+			log.Info(fmt.Sprintf("set the value %v for environment variable %v.", flags.AnsibleRolesPath,
+				aoflags.AnsibleRolesPathEnvVar))
```

Comment:
NIT: I would've used `log.Infof("set the value %v...`.  If you have to make changes for other comments, fix this while you are there. But if not, we can fix in a different PR as what you have is functional.